src/palace/manager/scripts/playtime_entries.py

```diff
@@ -238,7 +238,12 @@ def _fetch_report_records(self, start: datetime, until: datetime) -> Query:
                 sql_max(coalesce(PlaytimeSummary.title, "")).label("title2"),
                 count(distinct(PlaytimeSummary.loan_identifier)).label("loan_count"),
             )
-            .where(PlaytimeSummary.timestamp.between(start, until))
+            .where(
+                and_(
```

Comment:
This where clause is exclusive whereas between is inclusive.

Reply:
Good freakin catch.  I don't know how you do it.

Reply:
😊 Thanks!

The tests really caught it. I just noticed that we were needing to relax a test that should've passed.